tests/acceptance/features/apiSharingNg1/sharedWithMe.feature

```diff
@@ -5268,3 +5268,111 @@ Feature: an user gets the resources shared to them
     Then the HTTP status code should be "200"
     And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "NewSpace"
     And user "Brian" should not be able to download file "FolderToShare/lorem.txt" from space "Shares"
+
+  @env-config
+  Scenario Outline: update permission role of a shared folder from other roles to Denied (Personal Space)
```

Comment:
The scenario title says it is to update permission role to Denied but the scenario lists the shares after share role is updated to Denied. Update the title according to the scenario, and also the PR title.

Reply:
Wow. :+1: